src/Components/HomePage/QuizCard.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Heading } from "@chakra-ui/layout";
+import { Category, Quiz } from "../../Database/quiz.type";
+import { Link } from "react-router-dom";
+
+type QuizCardProps = {
```

Comment:
This type declaration can be moved to a `.types.ts` file

Reply:
Yes it can.. but I thought its only being used locally , hence kept it there. But I get your point